docs/generators/fsharp-functions.md

```diff
@@ -22,7 +22,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
-|sourceFolder|source folder for generated code| |OpenAPI/src|
+|sourceFolder|source folder for generated code| |OpenAPI\src|
```

Comment:
Not sure it is related to the PR.

Reply:
Well, it definitely doesn't.
But I run `./bin/utils/export_docs_generators.sh` as it's said in PR checklist, which lead to some docs changes.
Should I reverse those?

Reply:
I guess you are running the script from a windows environment. I suppose it is run on an unix environment on the CI so you should indeed revert them.

Reply:
ok, done.
Thank you for your response